.github/workflows/node-ci.yml

```diff
@@ -12,3 +12,38 @@ jobs:
       - uses: ./actions/check-for-changeset
       - uses: ./actions/shared-node-cache
       - run: yarn lint
+
+      # Test filter-files
+      - uses: ./actions/filter-files
+        id: fftest1
+        with:
+          changed-files: '["one", "two/three"]'
+          files: 'one, two, three'
```

Comment:
I know this is covered in the docs, but could you add a test case with the same `changed-files`, but with `files: 'two/three'` to illustrate that it matches exact file paths?